mrp_sale_info/tests/test_mrp_sale_info.py

```diff
@@ -67,6 +77,10 @@ def test_mrp_sale_info(self):
         self.assertEqual(production.sale_id, self.sale_order)
         self.assertEqual(production.partner_id, self.partner)
         self.assertEqual(production.client_order_ref, self.sale_order.client_order_ref)
+        self.assertEqual(self.sale_order.mrp_production_count, 1)
+        sale_action = self.sale_order.action_view_mrp_production()
+        self.assertEqual(sale_action["res_id"], production.id)
+        production.action_confirm()
```

Comment:
From what I understood, the `super()._prepare_procurement_values()` fixes a bug on native module sale_mrp (which does not make the right link between mrp.production and sale.order.line).

So I would add juste a test line testing this and remove the old code testing the native button.

```suggestion
        self.assertEqual(production.sale_line_id, self.sale_order.order_line)
```